bitnami/zookeeper/templates/statefulset.yaml

```diff
@@ -65,8 +65,8 @@ spec:
       {{- if .Values.priorityClassName }}
       priorityClassName: {{ .Values.priorityClassName }}
       {{- end }}
+      initContainers: {{ include "common.tplvalues.render" (dict "value" .Values.initContainers "context" $) | trim | nindent 8 }}
```

Comment:
In this case, if there are no initContainers specified in the _values.yaml_, an empty `initContainers:` section is going to be rendered in the statefulset, right?

I think what we are doing in other cases is something like
```yaml
{{- if or (and .Values.volumePermissions.enabled .Values.persistence.enabled) .Values.initContainers }}
initContainers:
  {{- if and .Values.volumePermissions.enabled .Values.persistence.enabled }}
  - name: volume-permissions
    ...
  {{- end }}
  {{- if .Values.initContainers }}
  {{- include "common.tplvalues.render" ( dict "value" .Values.initContainers "context" $ ) | nindent 8 }}
  {{- end }}
{{- end }}
```